### PR TITLE
chore: add volunteer ArkaPrabhaChowdhury

### PR DESCRIPTION
**Pull request by ArkaPrabhaChowdhury**

### CODE REVIEW

File: data/users/ArkaPrabhaChowdhury.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Arka Prabha Chowdhury",
+  "bio": "Front end developer | Intern at german startup",
+  "image": "https://www.linkpicture.com/q/me_6.jpg",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["ReactJS","Tailwind","Javascript"]
```
Comment (codewithvoid) [inline data/users/ArkaPrabhaChowdhury.json:8]: Move javascript to separate `programming` category like [here](https://github.com/codewithvoid/AskDevs/blob/main/data/users/codewithvoid.json)

File: data/users/ArkaPrabhaChowdhury.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Arka Prabha Chowdhury",
+  "bio": "Front end developer | Intern at german startup",
```
Comment (codewithvoid) [inline data/users/ArkaPrabhaChowdhury.json:3]: Just pick 1 title. Preferably, `Frontend Developer`
```json
"bio": "Frontend Developer"
```

File: data/users/ArkaPrabhaChowdhury.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://www.linkpicture.com/q/me_6.jpg",
```
Comment (codewithvoid) [inline data/users/ArkaPrabhaChowdhury.json:4]: optional: use github image url https://avatars.githubusercontent.com/u/92904175?v=4.

This loads well across geographic regions

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thank you @ArkaPrabhaChowdhury for volunteering. Left few comments. PTAL.

**Comment by ArkaPrabhaChowdhury** [comment]
Hey, @codewithvoid Could you please check it now?

File: data/users/ArkaPrabhaChowdhury.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Arka Prabha Chowdhury",
+  "bio": "Front end developer",
+  "image": "https://avatars.githubusercontent.com/u/92904175?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["ReactJS","Tailwind","Javascript"]
```
Comment (codewithvoid) [inline data/users/ArkaPrabhaChowdhury.json:8]: Remove javascript from here

Comment (codewithvoid) [inline data/users/ArkaPrabhaChowdhury.json:8]: nit: TailwindCSS?

File: data/users/ArkaPrabhaChowdhury.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Arka Prabha Chowdhury",
+  "bio": "Front end developer",
```
Comment (codewithvoid) [inline data/users/ArkaPrabhaChowdhury.json:3]: `Frontend Developer`

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by ArkaPrabhaChowdhury** [comment]
@codewithvoid My apologies, PTAL now.

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes @ArkaPrabhaChowdhury. Welcome!

**Pull request merged**